src/python_testing/TC_AccessChecker.py

```diff
@@ -148,6 +196,53 @@ def _record_errors(self):
                                       problem="Unknown attribute")
                     self.success = False
                     continue
+            # Check that we have information for all the required commands
+            for command_id in cmds[cluster_id]:
+                location = CommandPathLocation(endpoint_id=endpoint_id, cluster_id=cluster_id, command_id=command_id)
+                if command_id not in xml_cluster.accepted_commands.keys():
+                    self.record_warning(test_name="Access Checker", location=location,
+                                        problem="Cluster command not found in spec XML")
+                    continue
+                if command_id not in Clusters.ClusterObjects.ALL_ACCEPTED_COMMANDS[cluster_id]:
+                    self._record_error(test_name="Access Checker", location=location,
+                                       problem="Unknown command")
+                    self.success = False
+                    continue
+
+    async def _maybe_run_command_access_test_for_cluster_privilege(self, endpoint_id, cluster_id, device_cluster_data, xml_cluster: XmlCluster, privilege: Clusters.AccessControl.Enums.AccessControlEntryPrivilegeEnum):
+        """ Runs a command only if the required cluster privilege is HIGHER than the specified privilege. In this way,
+            no commands are actually run on the device, which means there are no side effects. However, we can differentiate
+            ACL rejections from commands being unsupported.
+        """
+        ota_exception = self.user_params.get('ci_only_linux_skip_ota_cluster_disallowed_for_certification', False)
+        if cluster_id == Clusters.OtaSoftwareUpdateRequestor.id and ota_exception:
+            logging.warn('WARNING: Skipping OTA cluster check for CI. THIS IS DISALLOWED FOR CERTIFICATION')
+            return
+
+        for command_id in checkable_commands(cluster_id, device_cluster_data, xml_cluster):
+            spec_requires = xml_cluster.accepted_commands[command_id].privilege
+            command = Clusters.ClusterObjects.ALL_ACCEPTED_COMMANDS[cluster_id][command_id]
+            location = CommandPathLocation(endpoint_id=endpoint_id, cluster_id=cluster_id, command_id=command_id)
+            name = f"Command test - privilege {privilege}"
+            if operation_allowed(spec_requires, privilege):
+                # In this test, we're only checking that the disallowed commands are rejected so that there are
+                # no side effects. Commands are checked with admin privilege in their cluster tests. The error that
+                # may be let through here is if the spec requires operate and the implementation requires admin.
+                continue
+            try:
+                timed = None
+                if command.must_use_timed_invoke:
+                    timed = 65535
```

Comment:
Why is it 65535 here? Why that specific number?

Reply:
It's the largest value for timed request. It doesn't really need to be that long, since we're sending the command right away, but I don't have any particular reason to choose another number, and I needed something. Added a comment explaining this is basically large and arbitrary.